index.js

```diff
@@ -57,6 +61,32 @@ module.exports.mime = mime
 var listenerCount = EventEmitter.listenerCount
   || function(emitter, type){ return emitter.listeners(type).length; };
 
+
+function responseStatus(res, code, msg) {
+  if(msg == null) {
+    msg = statuses[code]
```

Comment:
Do not reassign a variable tried into `arguments`, as that results in a v8 deopt. Declare a new variable name, please.